app/core/ingestion/pipeline.py

```diff
@@ -25,14 +23,6 @@ def generate_pipeline_ingest_input(db: Session) -> Sequence[DocumentParserInput]
         )
         .join(Organisation, Organisation.id == FamilyOrganisation.organisation_id)
         .join(Geography, Geography.id == Family.geography_id)
-        .filter(
-            Family.family_status.in_([FamilyStatus.PUBLISHED, FamilyStatus.CREATED])
-        )
-        .filter(
-            FamilyDocument.document_status.in_(
-                [DocumentStatus.CREATED, DocumentStatus.PUBLISHED]
-            )
-        )
```

Comment:
Looking at this has made me realise that we're going to need a corresponding update to the search query to avoid returning documents from a search that have been deleted...

Reply:
Working on this in separate PR